gst/nnstreamer/include/nnstreamer_plugin_api_filter.h

```diff
@@ -160,6 +160,16 @@ typedef struct _GstTensorFilterProperties
   int throughput; /**< The average throughput in the number of outputs per second */
 } GstTensorFilterProperties;
 
+/**
+ * @brief Structure definition for tensor-filter framework statistics
+ */
+typedef struct _GstTensorFilterFrameworkStatistics
+{
+  int64_t total_invoke_num;      /**< The total number of calls to invoke */
```

Comment:
how about adding flag to get the latency?
tensor-filter can set this flag true to calculate the latency. (default false to not get the latency)

Reply:
With the flag to enable the measuring of latency, do you also want to link it with PROP_LATENCY? The description of PROP_LATENCY is towards tensor filter measuring the latency than extension measuring the latency around the backend library invoke call.
If you think its alright to control this latency with PROP_LATENCY, I will update it.

Reply:
Let's do this later, not this PR.